datalad_container/extractors/metalad_container.py

```diff
@@ -0,0 +1,86 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""MetadataRecord extractor for files stored in Datalad's own core storage"""
+import json
+import logging
+import subprocess
+import time
+from uuid import UUID
+
+CURRENT_VERSION = "0.0.1"
+
+
+try:
+    from datalad_metalad.extractors.base import DataOutputCategory, ExtractorResult, FileMetadataExtractor
+    from datalad_metalad import get_file_id
+except Exception as e:
+    raise
```

Comment:
this is a placeholder for https://github.com/datalad/datalad-container/pull/200/files#r1129521291

Reply:
sorry -- this no longer leads anywhere specifically seems to me. So what is it for? may be just a comment instead of try/except would suffice for a placholder?

Reply:
The try/except was where I was planning on gating the plugin in case metalad isn't installed. 

Since it only has the 1 entry point, and if metalad is not installed the entry point fails before we get to this code, I dont think we need to do anything fancy with the imports.